packages/shared/src/UI/components/TokenIcon/index.tsx

```diff
@@ -38,7 +38,11 @@ export const TokenIcon = memo(function TokenIcon(props: TokenIconProps) {
         enabled: !logoURL,
         queryFn: async () => {
             const logoURLs = isNFT
-                ? await Hub.getNonFungibleTokenIconURLs(chainId, address)
```

Comment:
cc @guanbinrui this is a bug in the old code:

(main branch) `packages/web3-providers/src/Web3/Base/apis/HubNonFungibleAPI.ts` function `getNonFungibleTokenIconURLs` never implemented by any subclass but called.